deploy/roles/sas/tasks/main.yml

```diff
@@ -36,6 +38,24 @@
   tags:
     - sas
 
+- name: check sasenv script exists sas-console
+  stat:
+    path: "{{ sas_home }}/SASFoundation/9.4/bin/sasenv_local"
```

Comment:
@stirlingv Shouldn't we use `sas_version` here instead of 9.4?

Reply:
@tanderegg yup!